templates/default/cg_foodcritic_tests.rb.erb

```diff
@@ -47,12 +50,8 @@ rule 'CG001', 'Ensure all cookbook dependencies depend on a specific version' do
     name = ast.xpath(%q{//command[ident/@value='name']})
 
     if name.to_s.match('<%= @matches %>')
-      # Get and loop through the hash with dependencies
-      # and check if they depend on a specific version
       dependency_versions(ast).each do |name,version|
-        unless version.to_s != '' && /^(?:= )?\d+\.\d+\.\d+$/ === version.xpath(%q{@value}).to_s
-          matches << name
-        end
+        matches << name unless depends_on_specific_version(version)
```

Comment:
You are right!!

Please let me do this the right way. I'm editing it on github. 

I will do a checkout, and re-do this PR. Sorry!